mplc/doc/documentation.md

```diff
@@ -109,6 +110,16 @@ my_scenario = Scenario(partners_count=3,
                        minibatch_count=3)
 ```
 
+### Plot data distribution
+
+If you whish to visualize the partners' ground truth distribution of the train set per class (with and without corruption if there is any), you can use the method `plot_data_distribution()`. You may choose whether you want to display the plot straight away (`display` is `True` by default) or save it (`save` is `False` by default).
+
+```python
+my_scenario.plot_data_distribution(save=False, display=True)
+```
+
+> Note: the plot is automatically saved when calling the `run()` method described below.
+
```

Comment:
To provide an example to the reader, could you @kat-leen add an example graph here and/or add a link to the notebook?